data/presets/presets/barrier/guard_rail.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "maki-barrier",
+    "geometry": [
+        "vertex",
```

Comment:
Can it really be a vertex?  (node on a way)

Reply:
Whoops, I copied the preset from kerb and it specified vertex as valid, so that's where it's from. I didn't think to remove it or check against other barrier types. Pushed the fix.

Reply:
No problem- just making sure!